salt/metalk8s/orchestrate/apiserver.sls

```diff
@@ -0,0 +1,39 @@
+{%- set dest_version = pillar.metalk8s.cluster_version %}
```

Comment:
Given you're passing it from the call site, and that it may likely be useful in case of downgrade (I guess), maybe we should consume it:

```suggestion
{%- set dest_version = pillar.orchestrate.dest_version %}
```

Reply:
bad copy/paste. :)